docs/release/3.4/release-3-4-8.md

```diff
@@ -0,0 +1,83 @@
+OSG Software Release 3.4.8
+==========================
+
+**Release Date**: 2018-02-07
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   Critical GlideinWMS bug fix for heavily loaded frontends (<https://cdcvs.fnal.gov/redmine/issues/18748>)
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.8%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+!!! note "Notes"
+    -   OSG 3.4 contains only 64-bit components.
+    -   StashCache is supported on EL7 only.
```

Comment:
Are these necessary for this release? None of those packages have been touched.

Reply:
They still apply even if we haven't touched those specific packages, though.